Components/Card/Card.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="Card.css">
+  <title>Perry's Card</title>
+</head>
+
+<body>
+
+<div id="card">
+  <img src="images/20240413_095423.jpg" alt="An image of perry chamileke smiling in a cap">
+  <article>
```

Comment:
Good idea to use the article tag. But the structure is not correct. The article tag should be the first element that contains all other elements.  Because the image belongs to the article too